.github/workflows/issues.yml

```diff
@@ -5,16 +5,13 @@
   on:
     issues:
       types: [opened, reopened, closed]
-  permissions: 
-    issues: write
-    contents: read
   jobs:
     jira_task:
       name: Create Jira issue
       if: github.event.action == 'opened'
       runs-on: ubuntu-latest
-      permissions:
-        issues: write
```

Comment:
remove permissions are not needed any more?

Reply:
`permissions` are used to modify the default permissions granted to the GITHUB_TOKEN. Since we are not using it anymore, there's no point in setting the permissions